DTS-STN/c19-benefits-node
# C19BF-100 French Updates

# [C19BF-100 French Updates](https://jira-dev.bdm-dev.dts-stn.com/browse/C19BF-100be)

Updates to the french texts as requested on the documents included on the jira ticket 


## Test Instructions
1. Verify using the test link or  
2. download and run 

## Help Requested
- N/A

## Checklist
